src/mz/src/bin/mz/command/region.rs

```diff
@@ -30,6 +30,8 @@ pub enum RegionSubcommand {
     Enable {
         #[clap(hide = true, short, long)]
         version: Option<String>,
+        #[clap(hide = true, short, long)]
```

Comment:
OOC, why hide the flag? (This overall makes my life more difficult, since valid and necessary flags often don't show up in the help output!)

Reply:
We're planning to release the tool to end users soon, so we've marked as hidden all flags that are only relevant to Materialize developers.